versioned_docs/version-v2.7.0/releases.md

```
@@ -60,7 +60,7 @@ Release branches and PRs are managed as follows:
 - For critical fixes that need to be included in a patch release, PRs should always be first merged to master
   and then cherry-picked to the release branch. PRs need to be guaranteed to have a release note written and
   these descriptions will be reflected in the next patch release.
-  The cherry-pick process of PRs is executed through the script. See usage [here](https://project-hami.io/docs/contributor/cherry-picks).
+  The cherry-pick process of PRs is executed through the script. See usage [here](/docs/contributor/cherry-picks).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)